content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -79,51 +79,51 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
-## Installing runtime {#installing-runtime}
+## Installing a container runtime {#installing-runtime}
 
 To run containers in Pods, Kubernetes uses a
 {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
 
-{{< tabs name="container_runtime" >}}
-{{% tab name="Linux nodes" %}}
-
 By default, Kubernetes uses the
 {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
 to interface with your chosen container runtime.
 
 If you don't specify a runtime, kubeadm automatically tries to detect an installed
-container runtime by scanning through a list of well known Unix domain sockets.
-The following table lists container runtimes and their associated socket paths:
-
-{{< table caption = "Container runtimes and their socket paths" >}}
-| Runtime    | Path to Unix domain socket        |
-|------------|-----------------------------------|
-| Docker     | `/var/run/dockershim.sock`        |
-| containerd | `/run/containerd/containerd.sock` |
-| CRI-O      | `/var/run/crio/crio.sock`         |
-{{< /table >}}
-
-<br />
-If both Docker and containerd are detected, Docker takes precedence. This is
-needed because Docker 18.09 ships with containerd and both are detectable even if you only
-installed Docker.
-If any other two or more runtimes are detected, kubeadm exits with an error.
+container runtime by scanning through a list of known endpoints.
 
-The kubelet integrates with Docker through the built-in `dockershim` CRI implementation.
+If multiple or no container runtimes are detected kubeadm will throw an error
+and will request that you specify which one you want to use.
 
 See [container runtimes](/docs/setup/production-environment/container-runtimes/)
 for more information.
+
+This tables below include the known endpoints for supported operating systems:
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux" %}}
+
+{{< table >}}
+| Runtime      | Path to Unix domain socket                   |
+|--------------|----------------------------------------------|
+| containerd   | `unix:///var/run/containerd/containerd.sock` |
+| cri-o        | `unix:///var/run/crio/crio.sock`             |
+| Docker       | `unix:///var/run/cri-dockerd.sock`           |
```

Comment:
docker

Reply:
the name of the project is Docker, no?
https://en.wikipedia.org/wiki/Docker_(software)

containerd is lowercase:
https://containerd.io/

Reply:
I just wanted it to match what kubernetes says, like with `kubectl get nodes -o wide`.

Reply:
Suppose the company calls their product "Docker Engine", in case it actually matters

Reply:
ok, let's see if the docs reviewers want this to be changed any further.

Reply:
I'd use “Docker Engine” here.

Reply:
It says "Docker" now, I don't think anyone confused with the company or their VM offering

Reply:
Let's use “Docker Engine” here. There's going to be new focus on the difference between different things with Docker in their name.

Reply:
Or possibly:
```
Docker Engine (via `cri-dockerd`)
```

Reply:
I'm going with "docker". I'm sure that Docker Inc. will change their mind again, and I don't really care (since Moby)

https://deploy-preview-13359--kubernetes-sigs-minikube.netlify.app/docs/runtimes/

Need to document some of the internals, like docker-env and podman-env, so I don't think kubeadm/crictl will cut it.

The intention is for it to match the value of the `--container-runtime` parameter, and don't intend to change that.

Reply:
updating to `Docker Engine (using cri-dockerd)` and adding a brief note what it is.